packages/atomic/src/pages/examples/insights.html

```diff
@@ -36,8 +36,14 @@
         width: 100%;
         display: grid;
         grid-template-columns: 1fr auto auto auto;
+        /* grid-template-areas:
+          '1fr auto auto auto'
+          'tags tags tags tags'; */
+
         grid-gap: 0.5rem;
 
+        background: var(--atomic-neutral-light);
```

Comment:
Probably won't have time to tackle this, but it would have been nice to have `insight-layouts` components for the CSS in this page. The same way we have search layouts components.

Reply:
yeah, I have a task for that in JIRA... the plan is to basically transport these styles over to layout components